install_config/aggregate_logging.adoc

```diff
@@ -446,6 +454,9 @@ openshift_logging_es_pvc_dynamic value.
 create per Elasticsearch instance. For example, 100G. If omitted, no PVCs are
 created and ephemeral volumes are used instead. If you set this parameter, the logging installer sets `openshift_logging_elasticsearch_storage_type` to `pvc`.
 
+If no value for the `openshift_logging_es_pvc_dynamic` parameter has been set, you must set a value for this parameter.
+Read the description of `openshift_logging_es_pvc_prefix` for more information.
```

Comment:
@jhoyt-rh Should this say _If the `openshift_logging_es_pvc_dynamic` parameter has been set to false_ rather than no value to match the wording in `openshift_logging_es_pvc_dynamic`?

Reply:
Made this good change. THX